identity/conf/routes

```diff
@@ -63,4 +63,5 @@ GET         /email-prefs                            controllers.EditProfileContr
 POST        /email-prefs                            controllers.EditProfileController.saveEmailPreferencesAjax
 
 GET         /consent                                controllers.EditProfileController.displayConsentJourneyForm(journey: Option[String], consentHint: Option[String], newsletterHint: Option[String])
+POST        /repermissioned                         controllers.EditProfileController.submitRepermissionedFlag(returnUrl: String)
```

Comment:
Could returnUrl be in the post body instead?

Reply:
while i'm here, could this endpoint be /complete-consent? So i can just drive all users through it